web/src/components/gnd-feature-type-editor/gnd-form-editor.js

```diff
@@ -48,6 +53,23 @@ const styles = (theme) => ({
   },
 });
 
+const DragHandle = sortableHandle(() => <span>::</span>);
```

Comment:
For this I think we can use `drag_indicator` from https://www.npmjs.com/package/@material-ui/icons